contrib/oath-toolkit/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "oath-toolkit"
+pkgver = "2.6.11"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "gmake",
+    "automake",
+    "libtool",
+    "gtk-doc-tools",
+    "pkgconf",
+    "libxml2-progs",
+]
+makedepends = ["linux-pam-devel", "libxml2-devel"]
+pkgdesc = "OATH one-time password toolkit"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "GPL-3.0-or-later"
+url = "http://www.nongnu.org/oath-toolkit"
+source = f"http://download.savannah.gnu.org/releases/{pkgname}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "fc512a4a5b46f4c43ab0586c3189fece4d54f7e649397d6fa1e23428431e2cb4"
+# TODO: failing checks are yet to be investigated
+options = ["!check"]
+
+
+def post_install(self):
+    for lib in ["pskc", "oath"]:
+        self.install_man(f"lib{lib}/man/{lib}_*.3", glob=True)
+        self.install_man(f"{lib}tool/{lib}tool.1")
+
+
+@subpackage("oathtool")
+def _oathtool(self):
+    return ["usr/bin/oathtool", "usr/share/man/man1/oathtool.1"]
+
+
+@subpackage("pskctool")
+def _pskctool(self):
+    return ["usr/bin/pskctool", "usr/share/man/man1/pskctool.1"]
```

Comment:
hmm.. thinking again, is there a need to split these? if the whole thing is like just a few mb then there's usually not much point

Reply:
I thought it was good practice to split them, but I have no problem merging them either.

Reply:
splitting is generally to split up large stuff, or things that pull in some big dep chain (e.g. maybe some gui component), or auto policy stuff (default splits, devel , etc). but for small programs like this it's usually more hassle (entire thing is very small overall and having to hunt for a few binaries instead of installing just 1 thing is usually an extra annoyance).

so compared to e.g. alpine there is less granular splitting here per-program or whatever as it's not as final-size focused

Reply:
So I should get rid of this split and just keep the devel subpackage?

Reply:
yeah

Reply:
So, `oath-toolkit-progs` or should they just go into the main package?

Reply:
if they're 45kb then there's no point to split that imo